JS/Lesson19/esra/recursion.js

```diff
@@ -123,10 +185,73 @@ const nestedObjEx7 = {
 	},
 	h: 5,
 };
+
+function findMaxDepth(obj) {
+if (typeof obj !== 'object') {
+	return 0;
+}
+
+let maxDepth = 0;
+
+for (const key in obj) {
+	if (obj.hasOwnProperty(key)) {
+		const depth = findMaxDepth(obj[key]) + 1;
+		maxDepth = Math.max(maxDepth, depth);
+	}
+}
+
+return maxDepth;
+}
+
+const maxDepth = findMaxDepth(nestedObjEx7);
+console.log('Exercise7: ', 'Max Depth:', maxDepth);
+
 // 8. Write a recursive function to reverse the order of words in a sentence.
 const sentenceEx8 = 'Hello, how are you?';
+
+function reverseToWords(sentence) {
```

Comment:
'ReverseToWords' name is not correct English wise, 'reverseSentence' would be more appropriate.